packages/app/src/systems/Account/machines/accountsMachine.tsx

```diff
@@ -46,6 +50,8 @@ export type MachineEvents =
       type: 'ADD_ACCOUNT';
       input: string;
     }
+  | { type: 'EDIT_ACCOUNT'; input: string }
+  | { type: 'UPDATE_ACCOUNT_NAME'; input: AccountInputs['updateAccountName'] }
```

Comment:
I think here it would be possible to do everything in a single action; EDIT_ACCOUNT could receive the `name and address`.

Reply:
And maybe the overlay machine should have some params data?